### DISCUSSION TITLE
Getting Invalid Hook Error

### DISCUSSION THREAD

**Discussion by theashishmaurya** [facebook/docusaurus#8640] [Q&A]
I installed a fresh Docusaurus and got the below error. I haven't even changed anything yet. Can someone help?
![image](https://user-images.githubusercontent.com/50199029/217272283-65f58301-c1e2-41f6-b03e-74e08b8cd26c.png)

**Comment by slorber** [comment]
Please create a repro showing the problem.

--- 

This is likely an issue only you have. Maybe you tried to init Docusaurus inside a monorepo, or an existing npm project, and you have multiple versions of React in your project, like the error message says.

Try installing Docusaurus in another clean folder, or on another computer.

Most likely it's not a Docusaurus issue, but an issue you have because of your own setup.
